modules/ClinikalAPI/sql/0_1_0-to-0_2_0_upgrade.sql

```diff
@@ -149,3 +149,45 @@ ALTER TABLE `clinikal_templates_map`
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`) VALUES
 ('lists', 'clinikal_templates', 'Clinikal templates', 0, 0, 0, '', '', '', 0, 0, 1, '', 1);
 #EndIf
+
+
+DELETE FROM list_options where list_id="drug_route" OR option_id="drug_route";
```

Comment:
add #IfNotRow.... statement for all the lists
you can write  
#IfNotRow2D list_options list_id drug_route option_id per_oris for example

Reply:
I can not since I don't know what values are there in the list before .........
this is why I added DELETE